### ISSUE TITLE
rule-semicolon-newline-after should allow end-of-line comments

### ISSUE THREAD

**Issue by davidtheclark** [stylelint/stylelint#208]
Right now the following is complaining:

```css
.ht-Toggle {
  position: relative; /* 1 */
  height: var(--input-size-height); /* 9 */
  padding-top: calc(0.5 * (var(--input-size-height) - var(--lineheight-pxbase))); /* 9 */
}
```

I believe the reason is because of the comment between the semicolon and the end of line. Sooooo ... if we allow the comments after the semicolon, the rule *name* is not entirely accurate. But I think we could just point out this exception.
